sysboundary/sysboundary.cpp

```diff
@@ -146,151 +144,192 @@ bool SysBoundary::addSysBoundary(
+         if ((faces[2] || faces[3]) && isPeriodic[1]) {
+            if (myRank == MASTER_RANK)
+               cerr << "You set boundaries.periodic_y = yes and load Maxwellian system boundary conditions on the y+ "
+                       "or y- face, are you sure this is correct?"
+                    << endl;
          }
-         if((faces[4] || faces[5]) && isPeriodic[2]) {
-            if(myRank == MASTER_RANK) cerr << "You set boundaries.periodic_z = yes and load Maxwellian system boundary conditions on the z+ or z- face, are you sure this is correct?" << endl;
+         if ((faces[4] || faces[5]) && isPeriodic[2]) {
+            if (myRank == MASTER_RANK)
+               cerr << "You set boundaries.periodic_z = yes and load Maxwellian system boundary conditions on the z+ "
+                       "or z- face, are you sure this is correct?"
+                    << endl;
          }
-         if((faces[0] || faces[1]) && P::xcells_ini < 5) {
-            if(myRank == MASTER_RANK) cerr << "You load Maxwellian system boundary conditions on the x+ or x- face but there is not enough cells in that direction to make sense." << endl;
+         if ((faces[0] || faces[1]) && P::xcells_ini < 5) {
+            if (myRank == MASTER_RANK)
+               cerr << "You load Maxwellian system boundary conditions on the x+ or x- face but there is not enough "
+                       "cells in that direction to make sense."
+                    << endl;
             exit(1);
          }
-         if((faces[2] || faces[3]) && P::ycells_ini < 5) {
-            if(myRank == MASTER_RANK) cerr << "You load Maxwellian system boundary conditions on the y+ or y- face but there is not enough cells in that direction to make sense." << endl;
+         if ((faces[2] || faces[3]) && P::ycells_ini < 5) {
+            if (myRank == MASTER_RANK)
+               cerr << "You load Maxwellian system boundary conditions on the y+ or y- face but there is not enough "
+                       "cells in that direction to make sense."
+                    << endl;
             exit(1);
          }
-         if((faces[4] || faces[5]) && P::zcells_ini < 5) {
-            if(myRank == MASTER_RANK) cerr << "You load Maxwellian system boundary conditions on the z+ or z- face but there is not enough cells in that direction to make sense." << endl;
+         if ((faces[4] || faces[5]) && P::zcells_ini < 5) {
+            if (myRank == MASTER_RANK)
+               cerr << "You load Maxwellian system boundary conditions on the z+ or z- face but there is not enough "
+                       "cells in that direction to make sense."
+                    << endl;
             exit(1);
          }
+      } else {
+         if (myRank == MASTER_RANK)
+            cerr << "Unknown type of boundary read: " << *it << endl;
```

Comment:
Good, this is the essential change. And I gave up flagging all { } missing for if blocks above.